cmd/telemeter-server/main.go

```diff
@@ -86,6 +89,7 @@ func main() {
 		PartitionKey:       "_id",
 		Ratelimit:          4*time.Minute + 30*time.Second,
 		TTL:                10 * time.Minute,
+		MemcachedExpire:    24 * 60 * 60,
```

Comment:
24 hours seem a bit long. Are the JWTs valid for the same duration? I would have said 1 hour would be long already.

Reply:
That’s the current expiration time of the JWTs :/ so I think we should default to this for now just to keep traffic to tollbooth somewhat steady. We can of course override this with the flag at any point.